config/locales/en.yml

```diff
@@ -129,191 +104,5 @@ en:
 
       <p class="govuk-body">We do this for your security. We've deleted all the details you entered to protect your data.</p>
   coronavirus_form:
-    errors:
-      page_title_prefix: 'Error: '
-      heading: "There is a problem"
```

Comment:
Is it worth keeping these two? They seem pretty standard.

Reply:
Let's ship it, and we can always add them back.